rustls/src/crypto/aws_lc_rs/ticketer.rs

```diff
@@ -0,0 +1,354 @@
+use alloc::boxed::Box;
```

Comment:
How do you feel about making an intermediate commit that copies the ring ticketer into this file, so that this commit becomes a diff between the old ticketer and the new ticketer? Not sure how much code they would share but that's kind of the point.

Reply:
Done.